packages/universal/shared/README.md

```diff
@@ -1,15 +1,90 @@
-This package provides an extremely stable API for getting:
+# `@starbeam/shared`
 
-- The current timestamp as a number
-- The value of the `UNINITIALIZED` symbol
+**Unless you are very interested in the stability strategy of Starbeam, you don't
+need to understand how this package works.**
 
-Apps shouldn't use the exports of this dependency directly. Instead, separating the most fundamental
-parts of Starbeam's composition into a separate package allows two versions of Starbeam to coexist
-in the same process and **to share reactivity between them**.
+---
 
-In other words, if you access a Cell from version 1 of Starbeam in the context of a formula
-created in version 2 of Starbeam, updating the cell will invalidate the formula.
+This package is designed to make it possible for multiple copies of Starbeam to
```

Comment:
so long as they share a single `@starbeam/shared`, yeah?

Reply:
Nope! Multiple copies of `@starbeam/shared` coordinate to create only a single global symbol. The functions exported by `@starbeam/shared` always use the values under the symbol, so they don't need to be literally the same function or share the same module state.